snippets/odoo-python-snippets.cson

```diff
@@ -93,61 +93,62 @@
 	'Compute method':
 		'prefix': 'oo_method_compute'
 		'body': """
-@api.one
-@api.depends('$1', ...)
-def _compute_field(self):
-	self.field = $2
-		"""
+			@api.multi
```

Comment:
Changing the indentation is not necessary. If the first line is at the right indent, the other lines will follow.